src/main/java/uk/co/qualitycode/utils/functional/ArrayIterable.java

```diff
@@ -40,7 +40,7 @@ public void remove() {
     }
 
     /**
-     * Factory method to create a new <tt>ArrayIterable</tt> given an array of <tt>T</tt>.
+     * Factory method to create a new ArrayIterable given an array of T.
```

Comment:
Everything I changed in java is java doc, when you try to publish it tries to render it all and threw a billion errors, so these are the fixes to make it happy. I dont think i was wrong with any of my updates.